pkg/validation/internal/operatorhub.go

```diff
@@ -69,6 +69,12 @@ var validCategories = map[string]struct{}{
 	"Streaming & Messaging":  {},
 }
 
+var validInfraFeatures = map[string]struct{}{
+	"Disconnected": {},
```

Comment:
@gallettilance based on the latest email in pm sent by @jmccormick2001 then, we might also allow disconnected too. Also, would that be valid for the others options as well. WDYT @jmccormick2001?

Reply:
correct, my understanding is that all the values specified in the current docs (all lowercase) are valid within console and that the older style of spellings are also acceptable for now.